app/models/adjustment.rb

```diff
@@ -0,0 +1,30 @@
+class Adjustment < ApplicationRecord
+  delegate :value, to: :financial_transaction
+  delegate :financial_quarter, to: :financial_transaction
+  delegate :financial_year, to: :financial_transaction
+
+  belongs_to :financial_transaction,
+    class_name: "Transaction",
+    foreign_key: :transaction_id
+
+  include HasFinancialQuarter
+
+  belongs_to :user
+
+  has_one :report, through: :financial_transaction
+  has_one :activity, through: :financial_transaction, source: :parent_activity
+
+  validates :comment, presence: true
+
+  def value=(big_decimal)
+    financial_transaction.value = big_decimal
+  end
+
+  def financial_quarter=(integer)
```

Comment:
Is there a way to automatically delegate the setters, too? I feel like we're losing some of the benefits of delegation if we have to separately remember to define the setters.